src/pages/workspace/WorkspaceNewRoomPage.tsx

```diff
@@ -163,25 +126,25 @@ function WorkspaceNewRoomPage(props) {
 
     useEffect(() => {
         if (policyID) {
-            if (!_.some(workspaceOptions, (opt) => opt.value === policyID)) {
+            if (!workspaceOptions.some((opt) => opt.value === policyID)) {
                 setPolicyID('');
             }
             return;
         }
-        if (_.some(workspaceOptions, (opt) => opt.value === props.activePolicyID)) {
-            setPolicyID(props.activePolicyID);
+        if (!!activePolicyID && workspaceOptions.some((opt) => opt.value === activePolicyID)) {
+            setPolicyID(activePolicyID);
         } else {
             setPolicyID('');
         }
-    }, [props.activePolicyID, policyID, workspaceOptions]);
+    }, [activePolicyID, policyID, workspaceOptions]);
 
     useEffect(() => {
-        if (!(((wasLoading && !props.formState.isLoading) || (isOffline && props.formState.isLoading)) && _.isEmpty(props.formState.errorFields))) {
+        if (!(((wasLoading && !isLoading) || (isOffline && isLoading)) && isEmptyObject(errorFields))) {
             return;
         }
         Navigation.dismissModal(newRoomReportID);
         // eslint-disable-next-line react-hooks/exhaustive-deps -- we just want this to update on changing the form State
-    }, [props.formState]);
+    }, [isLoading, errorFields]);
```

Comment:
I am working on a PR and I am seeing a lint failure in this file: https://github.com/Expensify/App/actions/runs/8103662907/job/22148730565?pr=37556#step:4:19
> Warning:   147:8  warning  React Hook useEffect has missing dependencies: 'isOffline', 'newRoomReportID', and 'wasLoading'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

Questions are:
1. Should I include those other variables like it is suggesting?
2. Why is this failing lint now and not when this PR was merged?

Reply:
```
        // eslint-disable-next-line react-hooks/exhaustive-deps 
```
@iwiznia because you removed this comment in your PR

https://github.com/Expensify/App/pull/37556/files#diff-80df02834ebf2f103e25089bc976474c23f483a5c1188f54eb1f48f847860849L147

Reply:
Oh thanks! 🤦 
For some reason my IDE was marking that line as an error... thanks!